plugins/javadoc/src/test/kotlin/org/jetbrains/dokka/javadoc/JavadocModuleTemplateMapTest.kt

```diff
@@ -77,4 +78,65 @@ internal class JavadocModuleTemplateMapTest : AbstractJavadocTemplateMapTest() {
             assertEquals(DRI("com.test.package1"), assertIsInstance<RowJavadocListEntry>(list[1]).link.dri.single())
         }
     }
+
+    @Test
+    fun `single class with package documentation (kotlin)`() {
```

Comment:
It seems to be about module documentation?
```suggestion
    fun `single class with module documentation (kotlin)`() {
```